spec/features/hub/authenticate_spec.rb

```diff
@@ -62,7 +62,7 @@
     # Expect subject to
     expect(email.subject).to eq("Reset password instructions")
 
-    html_body = email.body.parts[1].decoded
+    html_body = email.body.encoded
```

Comment:
I'm surprised you need to use the encoded version of the email, rather than the decoded version here. I would expect the decoded version to have a valid link and the encoded one maybe not to.

Not a blocker, but curious what motivated the change.